bot/constants.py

```diff
@@ -618,6 +618,17 @@ class _Icons(EnvConfig):
 Icons = _Icons()
 
 
+class _GithubModsRepository(EnvConfig):
+    section = "github_mods_"
+
+    owner: str = "python-discord"
+    name: str = "mods"
```

Comment:
I very vaguely remember the discussion behind this feature so I just want to double check whether the issue has to be raised on the `python-discord/mods` repo or `python-discord/admins` (?) repo.

Whenever a vote has passed the upvote threshold, the admins add a ticket to the admin tasks tracker** and not `python-discord/mods` repo- which is why I'm asking for clarification on which repo stale reviews should be posted.


**I'm not actually sure what the "admin tasks tracker" is, I always assumed it's a bunch of issues raised on `python-discord/admins` or something, similar to the `python-discord/meta` repo.

Reply:
Okay, admin tasks tracker is the `admin-tasks` repo.

Reply:
Yeah, the issue is to be raised on `python-discord/admin-tasks`, Chris just confirmed.